rust/arrow/src/error.rs

```diff
@@ -70,4 +71,12 @@ impl From<::std::string::FromUtf8Error> for ArrowError {
     }
 }
 
+impl Display for ArrowError {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        write!(f, "Arrow error happened!")
```

Comment:
Why not display the type of and details ArrowError?

Reply:
Details should be displayed in debug mode.

Reply:
I still don't understand, but perhaps it's because I'm not familiar with the Display trait.

Reply:
Yea, I'm with @nevi-me here.  His point is that "Arrow error happened!" is not that informative.  You would also want to know the variant of the enum and any payload, right?

Reply:
fixed.